gatsby-node.js

```diff
@@ -57,10 +57,7 @@ exports.onCreateWebpackConfig = ({ actions, plugins }) => {
       plugins.define({
         "process.env": {
           NODE_ENV: JSON.stringify(process.env.NODE_ENV),
-          IS_BETA: JSON.stringify(process.env.IS_BETA),
-          CONTEXT: JSON.stringify(process.env.CONTEXT),
-          URL: JSON.stringify(process.env.URL),
-          DEPLOY_PRIME_URL: JSON.stringify(process.env.DEPLOY_PRIME_URL),
+          AMPLITUDE_KEY: JSON.stringify(process.env.AMPLITUDE_KEY),
```

Comment:
Gatsby has some tricky behavior with environment variables, so it's simplest just to use them directly where needed. I wanted to keep them isolated to `constants/env.js`, but I ran into issues with that. Using them directly also allows dead code elimination to slightly reduce the size of the bundle, so that's nice.